src/sass/components/_position.scss

```diff
@@ -1,7 +1,59 @@
+// Position Item
+
+@include media-object('.position-item', '.position-item__avatar', '.position-item__content');
+
 .position-item {
-  .position-source {
-    color: rgba(171,171,171, 1);
-    font-size: 0.9em;
-    font-weight: 200;
-  }
+  position: relative;
```

Comment:
I'm okay with breaking up those 'responsibilities' further as it makes sense, but I'd prefer to not do it with utility classes. I would like to eventually get to a more generic pattern, but am okay with having some redundancy of styling between different components in the mean time. This is partly because I'm still wrapping my head around the way things are broken up, assuming there is some degree of refactoring of components that will need to happen anyway (based on brief talks with @DaleMcGrew), and that it still seems like an (iterative) step forward.

@pertrai1 I do still want to move towards more abstraction of patterns and styling, but find this a helpful step. If you think it would be better to adopt more abstraction from the jump, let me know.